cmake/OpenCVFindProtobuf.cmake

```diff
@@ -6,6 +6,7 @@ if(NOT WITH_PROTOBUF)
   return()
 endif()
 
+
```

Comment:
I'm sorry I have small mistake that is inserting unnecessary blank line.I had created pull request to fix it.